addon/mixins/map-route-metadata-ids-handler.js

```diff
@@ -0,0 +1,34 @@
+/**
+  @module ember-flexberry-gis-dummy
+*/
+
+import Ember from 'ember';
+
+/**
+  Map edit form mixin which handles metadata ids.
+  @class MapRouteMetadataIdsHandlerMixin
+*/
+export default Ember.Mixin.create({
```

Comment:
Name does not really corespond to mixin's usage. It does not handle anything.

Reply:
Fixed